protocols/kad/src/protocol.rs

```diff
@@ -40,6 +40,8 @@ use std::{io, iter};
 use unsigned_varint::codec;
 use wasm_timer::Instant;
 
+pub(crate) const DEFAULT_PROTO_NAME: Cow<'static, [u8]> = Cow::Borrowed(b"/ipfs/kad/1.0.0");
```

Comment:
```suggestion
pub const DEFAULT_PROTO_NAME: Cow<'static, [u8]> = Cow::Borrowed(b"/ipfs/kad/1.0.0");
```
Would it not make sense to export this constant from the crate? Just an idea, since its not necessary we don't need to do that at the moment.